### ISSUE TITLE
Error "TypeError: Cannot destructure 'options' as it is undefined" thrown when attempting to create a training call to a private model

### ISSUE THREAD

**Issue by AlexStan0** [replicate/replicate-javascript#82]
## Description

The trainModel function is throwing a "TypeError: Cannot destructure 'options' as it is undefined" error, even though the options object is passed as an argument to the function.

### Code Sample
```javascript
trainModel = async (filePath, instancePrompt, classPrompt, ...maxTrainSteps) => {
    // Create the training options for the training call
    let inputOptions = this.#createTrainingOptions(filePath, instancePrompt, classPrompt, ...maxTrainSteps);

    // Create a string to store the username of the model
    let callDestination = `${this.#username}/${this.#model}`;

    let options = {
        version: this.#trainerVersion,
        destination: callDestination,
        input: inputOptions,
    };

    // Create a training call to the model
    const response = await this.#replicate.trainings.create(options);

    console.log(response);

} // End trainModel()
```
The input options arg looks like this:

```javascript
{
            "instance_prompt": instancePrompt,
            "class_prompt": classPrompt,
            "instance_data": "{file url for training data}",
            "max_train_steps": 500
}
```

## Expected Behavior
The options object should be successfully passed to the trainings.create method without any errors.

## Additional Context
The code snippet lacks clear examples or instructions on what parameters should be passed to the trainings.create function. As a result, the API call [here](https://replicate.com/blog/dreambooth-api) is being used as a reference for the required parameters.

**Comment by AlexStan0** [comment]
Just looked through the code that creates the POST request for the training

```javascript
async function createTraining(model_owner, model_name, version_id, options) {
  const { ...data } = options;

  const training = this.request(`/models/${model_owner}/${model_name}/versions/${version_id}/trainings`, {
    method: 'POST',
    data,
  });

  return training;
}
```

For some reason it requires three other parameters that are not at all listed in the documentation, those being `model_owner`, `model_name`, and `version_id`. Although this in of itself is not too weird, the fact that it was not at all mentioned in the documentation is incredibly concerning as it seems those parameters are basically required for the `trainings.create()` function to work.

**Comment by mattt** [comment]
Hi @AlexStan0. Thanks for writing this up. The documentation in the README was out of sync with the actual implementation, but that's now fixed by #83.

For context, training / fine-tuning models is currently in private beta (see [this guide](https://replicate.com/docs/guides/fine-tune-a-language-model) for details). DreamBooth predates trainings, and uses a different API that's not supported by this library. Please follow the instructions from [this blog post](https://replicate.com/blog/dreambooth-api) to create custom models with DreamBooth.
